addons/account_peppol/wizard/peppol_config_wizard.xml

```diff
@@ -43,7 +47,7 @@
                         </field>
                     </div>
                 </group>
-                <group string="Danger zone" invisible="account_peppol_proxy_state not in ('sender', 'smp_registration', 'receiver')">
+                <group string="Danger zone" invisible="account_peppol_proxy_state not in ('sender', 'receiver')">
```

Comment:
I don't see why we should prevent people waiting to be receivers (that are already sender) to unregister from our AP. Same for updating their contact email ? @pmax-x

Reply:
@antoine162 discussed with PMAX, he agreed it makes no sense, let's just keep the new informative message for `smp_registration` users, but remove all the unecessary limitations of actions.